code/obj/railing.dm

```diff
@@ -150,10 +150,13 @@
 	attack_hand(mob/user)
 		src.try_vault(user)
 
+	attack_ai(mob/user)
+		return src.attack_hand(user)
+
```

Comment:
Needs either appropriate range checks, or just drop this in favor of just the `Bumped()` support

Reply:
I thought it was okay to do because the action bar has range checks but I can add them there too if you'd like

Reply:
ok nvm it is not okay to do I see what you mean lol